src/functions/parseSdJwtCredential.ts

```diff
@@ -32,6 +42,10 @@ export const parseSdJwtCredential = async (credential: string | object): Promise
 				}
 
 			}
+
+			return {
+				beautifiedForm: parseJwt(credential),
```

Comment:
This should probably be moved to a separate parser for the jwt_vc_json format, please advise.